build.gradle

```diff
@@ -1,36 +1,94 @@
 plugins {
-    id "com.jfrog.bintray" version "1.1"
-    id "com.github.jruby-gradle.base" version "0.1.5"
     id "java"
     id "checkstyle"
+    id "maven-publish"
+//    id "signing"
+    id "org.embulk.embulk-plugins" version "0.4.2"
 }
-import com.github.jrubygradle.JRubyExec
 repositories {
     mavenCentral()
-    jcenter()
-}
-configurations {
-    provided
 }
 
+group = "io.github.llibra"
 version = "1.5.0"
```

Comment:
Maybe nice to upgrade to the next `1.6.0` ?

Reply:
Thanks. Updated to `1.6.0`